magento/magento2
# Improvement sticky navbar

### Description (*)

Continue work that was done by @divyajyothi5321 in #27050 after long inactivity
Add my additionals and validate code follow less standards

This PR bring styles change make main navigation menu sticky when scroll down when page height enough in theme luma
#1 Add new sticky lib less in magento ui library (can reusable later)
#2 Apply sticky for navigation in luma theme. For maintain compatible i decided should not add sticky in blank theme at the moment

### Related Pull Requests

https://github.com/magento/magento2/pull/27050
https://github.com/magento/magento2/pull/26447
### Fixed Issues (if relevant)

1. Fixes magento/magento2#26135

### Manual testing scenarios (*)

1. Set theme to luma
2. Go to backend and create some item category for menu 
3. Go to frontend 
4. Scroll down to see menu sticky while scroll down

### Questions or comments


### Contribution checklist (*)
